### DISCUSSION TITLE
Feedback: wasted space on Saved Replies page

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [community/community#21126] [Other Feature Feedback, Questions, & Ideas]
Simple question: Can you improve this a bit?
![image](https://user-images.githubusercontent.com/65828559/180606555-09b70a58-c790-499a-8147-7ab3a06e9fdf.png)

**Comment by alieldeba** [comment]
what do you want me to improve ??
here you have all your replies and you can edit or delete it

the ... dots -is for the text truncate case Github can only show 35 letter

**Comment by Jay-o-Way** [comment]
Yes the dots is what i mean. Don't know why that 35 limit exists, but it's pretty unpractical.
